client/wfnews-war/src/main/angular/src/app/components/public-incident-page/incident-info-header/incident-header-panel.component.html

```diff
@@ -58,19 +58,16 @@ <h4>Active Evacuation Orders and Alerts</h4>
       </div>
     </div>
     <div *ngIf="isMobileView()">
-      <div class="header-content">
-        <div class="item">
-          <button mat-button class="mobile-button">
-            <mat-icon svgIcon="bookmark"></mat-icon>
+      <div class="contact-buttons">
+        <button mat-button class="mobile-buttons">
+            <img class="icon-contact" height="24" width="24" src="/assets/images/svg-icons/bookmark.svg"
```

Comment:
we can get rid of height="24" width="24"  from tag now since the same value has been declared in css class.

Reply:
Removed now, thanks for catching that!